### PR TITLE
Support for building latest quicly version

### PR DESCRIPTION
**Pull request by pbanicev**
This fixes building issues with latest quicly version.
Need to be run tested with some typical setup, see #334

### CODE REVIEW

**Comment by KeithWiles** [comment]
One of the actions failed for clang-format please click the details link and correct the problem before you convert the PR to a non-draft status. Thanks.

**Comment by KeithWiles** [comment]
Looks good, did you want to convert this to non-draft version and I can approve and merge. Thanks

File: examples/cnet-quic/meson.build [hunk 1]
```diff
@@ -56,7 +57,7 @@ if quicly_dep.found()
             quic_cflags += arg
         endif
     endforeach
-    quic_cflags += ['-I'+quicly_path+'/include', '-I'+picotls_path+'/include']
+    quic_cflags += ['-I'+quicly_path+'/../include', '-I'+picotls_path+'/../include']
```
Comment (maryamtahhan) [inline examples/cnet-quic/meson.build:60]: This change doesn't work if you follow the readme to build the quicly and picotls dirs

Comment (pbanicev) [reply examples/cnet-quic/meson.build:60]: Needed this to get it build locally, but can remove if it breaks compilation in your case.

**Comment by KeithWiles** [comment]
Is this one ready to merge and move out of draft status?

**Comment by pbanicev** [comment]
I think another pull request toke changes from here with additional commits/fixes on top. so one can close this draft PR once that PR is merged.

**Comment by maryamtahhan** [comment]
Yeah sorry - I pulled this into https://github.com/CloudNativeDataPlane/cndp/pull/343. We should be good to close this one.

**Comment by maryamtahhan** [comment]
Integrated into https://github.com/CloudNativeDataPlane/cndp/pull/343.

**Pull request closed without merging**